Dockerfile

```diff
@@ -1,24 +1,20 @@
 FROM golang:alpine as builder
 
 RUN apk --update --no-cache add make git g++ linux-headers
-# DEBUG
-RUN apk add busybox-extras
```

Comment:
just out of curiosity - do you know what we need this package for in headerSync?

Reply:
Which package are you referring to? I removed `busybox-extras` bc it didn't seem necessary

Reply:
Yeah, `busybox-extras` - I just wasn't sure what it was even. But I'll up for ✂️ it if it's not needed.

Reply:
I think it's [helpful tools](https://hub.docker.com/_/busybox) for tinkering inside the container. May be worth re-adding if we're `sh`ing in there a lot to mess around, but I'm hoping we can set-it-and-forget-it with minimal interference